recipes/moc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.12)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
```suggestion
conan_basic_setup(TARGETS)
```

Reply:
Since I want to do the global variable approach, I think keeping it as is should work. Correct?